Resources/Prototypes/Entities/Structures/Furniture/bench.yml

```diff
@@ -0,0 +1,70 @@
+- type: entity
+  name: comfortable bench
+  id: BenchComfSolo
+  parent: SeatBase
+  description: A bench with an extremely comfortable backrest.
+  components:
+  - type: Transform
+    anchored: true
+  - type: Rotatable
+  - type: Sprite
+    sprite: Structures/Furniture/Bench/comf_bench.rsi
+    layers:
+    - state: bench_solo_base
+      color: "#767e82"
+    - state: bench_solo_cover
+  - type: Physics
+    bodyType: Static
+    canCollide: true
```

Comment:
Isn't this already default to true.

Reply:
Tests in the past PR swear by it until I set it up like this.